### PR TITLE
Add test data

### PR DESCRIPTION
**Pull request by asl3**
Add additional wiki_sql test data to `test_sample.jsonl`

### CODE REVIEW

File: benchmark/wikisql/wiki_sql.py [hunk 1]
```diff
@@ -133,6 +135,8 @@ def get_tables_and_questions(source_file):
     errors = 0
 
     # Create sql query for each question and table
+    error_file = open("data/error_file.txt", "w")
```
Comment (gengliangwang) [inline benchmark/wikisql/wiki_sql.py:138]: let's make it `mismatched_results.txt`.

File: benchmark/wikisql/data/test_sample.jsonl
Comment (gengliangwang) [inline benchmark/wikisql/data/test_sample.jsonl:1]: Let's have a follow-up to increase to test input as 800 or 1000

File: pyspark_ai/prompt.py [hunk 2]
```diff
@@ -87,7 +87,7 @@
 Observation: 01-01-2006
 Thought: The correct Birthday filter should be '01-01-2006' because it is semantically closest to the keyword.
 I will use the column 'Birthday' to filter the rows where its value is '01-01-2006' and then select the COUNT(`Student`) 
-because COUNT gives me the total number of students.
+because the question asks for the total number of students.
```
Comment (gengliangwang) [inline pyspark_ai/prompt.py:90]: QQ: does this fix all mismatched all `COUNT` queries?

Comment (asl3) [reply pyspark_ai/prompt.py:90]: yes! except for maybe a couple, but overall yes

Comment (gengliangwang) [reply pyspark_ai/prompt.py:90]: This is great!

**Review by gengliangwang — APPROVED** [review]

**Pull request merged**